src/common/pjrt_implementation/module_builder/compile_options.cc

```
@@ -23,7 +24,14 @@ CompileOptions CompileOptions::parse(
           .value_or(false);
   options.backend = internal::parseBackendOption(compile_options, "backend")
                         .value_or(Backend::Default);
-
+  options.dump_inputs = internal::parseBoolOption(
+      compile_options,
+      "dump_inputs"); // nonexistent value is handled differently based on other
+                      // options, therefore no explicit defaulting with
+                      // value_or.
+  options.export_path =
+      internal::parseStringOption(compile_options, "export_path")
+          .value_or("codegen_solution");
```

This sets the default value of the export path? I'm thinking if it might be better to error out rather than set a default path. If we use a default path, I'm afraid of users overwriting their previous solution unintentionally, whereas having to set a path would proc the user to do it and make it less likely that they'd unintentionally overwrite. Thoughts?